components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -144,13 +144,7 @@ export abstract class AbstractTypeORMWorkspaceDBImpl implements WorkspaceDB {
             // We need to put the subquery into the join condition (ON) here to be able to reference `ws.id` which is
             // not possible in a subquery on JOIN (e.g. 'LEFT JOIN (SELECT ... WHERE i.workspaceId = ws.id)')
             .leftJoinAndMapOne('ws.latestInstance', DBWorkspaceInstance, 'wsi',
-                `wsi.id = (
-                    SELECT i.id
-                    FROM d_b_workspace_instance AS i
-                    WHERE i.workspaceId = ws.id
-                    ORDER BY i.creationTime DESC
-                    LIMIT 1
-                )`
+                `wsi.id = (SELECT i.id FROM d_b_workspace_instance AS i WHERE i.workspaceId = ws.id ORDER BY i.creationTime DESC LIMIT 1)`
```

Comment:
```
Request getWorkspaces failed with message: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ORDER BY wsiRunning.workspaceId DESC, GREATEST(`ws`.`creationTime`, `wsi`.`creat' at line 1
```
maybe that's not related to this line 🤷🏻‍♂️ 
just got it after creating a new team and visiting the empty workspaces page for that team.